includes/class-sensei-settings.php

```diff
@@ -881,6 +897,29 @@ public function render_learning_mode_setting( $args ) {
 		<?php } ?>
 		<?php
 	}
+
+	/**
+	 * Learning mode setting.
+	 *
+	 * @param array $args The field arguments.
+	 */
+	public function render_woo_commerce_settings( $args ) {
+		$image_path_desktop = Sensei()->assets->get_image( 'purchase-sensei-pro-desktop.png' );
+		$image_path_mobile  = Sensei()->assets->get_image( 'purchase-sensei-pro-mobile.png' );
+		?>
+		<div class="woocommerce-banner">
```

Comment:
Do you think it would be better to prefix all classes with `sensei` to avoid possible conflicts?

Reply:
Now thinking about it, maybe you should remove `woocommerce` from the class name because this banner is not related to `woocommerce`. I think the same banner is used on the Content Drip settings section as well.

Reply:
@donnapep we can do it that way, but it is essentially the same as what I do now, it will only move the logic from the $sections and generate $feilds to this new function (Add tabs). I don't have anything against it. just that everything will stay the same, I would still need to hide the Button and the Header for settings.

Reply:
@m1r0 regarding the renaming it to generic banner is something I wanted to do, but I am not 100% sure what will I need to change from content drip (it looks a bit different) so I decided to leave the renaming for that task, once I know what will be the same and what will be custom for the banners. So if you agree I will leave it for content drip

Reply:
> we can do it that way, but it is essentially the same as what I do now, it will only move the logic from the $sections and generate $feilds to this new function (Add tabs).

I'm less concerned with using similar code and more concerned with how Sensei and WCPC work together here when they are both installed. For example, using `master` branch and with the [WCPC plugin](https://github.com/automattic/sensei-wc-paid-courses) also activated, this is what WooCommerce settings looks like (because WCPC is adding this tab):

![Screen Shot 2022-02-14 at 9 22 40 AM](https://user-images.githubusercontent.com/1190420/153883204-daf0bc8d-c592-4ce0-8165-75be8c45b23d.jpg)

After switching to this branch, the previous setting is broken and no longer shows the label:

![Screen Shot 2022-02-14 at 9 20 30 AM](https://user-images.githubusercontent.com/1190420/153883256-56174f25-8219-4133-bfd1-21daad68db2b.jpg)

We'll need to figure out how to get both plugins to display their individual bits here so that everything works. Right now they seem to be stepping on each other's toes.

Reply:
Soo @donnapep I fixed the issue since it was a deal-breaker and I wasn't too happy with my decision how this was built before anyway.

Here is my new commit with the changes : [Commit](https://github.com/Automattic/sensei/pull/4801/commits/a4532eb75b54fd39e135e3488a725c92803af31b)

And here is how it looks now: 

| No plugin | With plugin |
| -- | -- |
| <img width="1512" alt="image" src="https://user-images.githubusercontent.com/7208249/154047680-72c8324b-87d9-4754-849d-ca63c4c10996.png"> | <img width="1512" alt="image" src="https://user-images.githubusercontent.com/7208249/154047773-efc3837e-0e98-4a9a-84d7-cf758f2164bd.png"> |